spyrmsd/parallel.py

```diff
@@ -0,0 +1,127 @@
+try:
+    from pebble import ProcessPool
+except ImportError:
+    errmsg = (
+        "Parallel execution of SPyRMSD (`prmsdwrapper`) requires `pebble`."
+        "Please install `pebble`: https://github.com/noxdafox/pebble"
+    )
+    raise ImportError(errmsg)
+
+import os
+import warnings
+from concurrent.futures import TimeoutError
```

Comment:
```suggestion
```
This is no longer needed, is it?

Reply:
Ooof, I totally missed this. I think distinguishing between the errors is good, I'll resolve this conversation.